### DISCUSSION TITLE
How to keep data structure comments?

### DISCUSSION THREAD

**Discussion by xdliyushen** [j178/leetgo#109] [Q&A] [answered]
### Verify steps

- [X]  我已经在 [Issue Tracker](https://github.com/j178/leetgo/issues) 中找过我要提出的请求 I have searched on the [Issue Tracker](https://github.com/j178/leetgo/issues) for a related feature request. 
- [X]  我已经仔细看过 [Documentation](https://github.com/j178/leetgo/blob/master/README.md) 并无法自行解决问题 I have read the [Documentation](https://github.com/j178/leetgo/blob/master/README.md) and was unable to solve the issue. 

### Description

Wishing there was a data structure comment that uses in the code inserts before the document.
suck like this in problem [25.reverse nodes in k-group](https://leetcode.com/problems/reverse-nodes-in-k-group/)
```js
/**
 * Definition for singly-linked list.
 * function ListNode(val, next) {
 *     this.val = (val===undefined ? 0 : val)
 *     this.next = (next===undefined ? null : next)
 * }
 */
```

**Comment by j178** [comment, marked as answer]
Thank you for opening this!

These comments are deleted intentionally by `removeUselessComments` modifier, if you need them, you can remove `removeUselessComments` from `code.modifiers` in your config file(located in ~/.config/leetgo/config.yaml).

**Comment by xdliyushen** [comment]
Thanks! It works!
